Package/Windows/ClientFiles.wxs

```diff
@@ -465,6 +465,24 @@
       <Component Id="System.Diagnostics.DiagnosticSource.dll" Guid="1E23AA5D-D9D4-4C0C-8A8E-714E5F952CF3">
         <File Id="System.Diagnostics.DiagnosticSource.dll" Source="$(var.Xamarin.Interactive.Client.Windows.TargetDir)\System.Diagnostics.DiagnosticSource.dll" />
       </Component>
+      <Component Id="Humanizer.dll" Guid="5A15CE3C-86F1-4DDA-B1A8-0659D1E0E20F">
```

Comment:
What's bringing in Humanizer and these SQLitePCLRaw assemblies? Specifically with Humanizer it brings in a ton of localization resources/assemblies as well. I would love to avoid it as a dependency.

Reply:
Roslyn 2.9. If you like, we can not bump Roslyn. The version we already had in this branch supports 7.3. Roslyn 2.9 is just bugfixes, and I don't actually know if we care about any of them.

Reply:
I assume there will be another Roslyn release that corresponds to VS 15.9 (2.9 corresponds to 15.8), but it might not be available from nuget.org in a timely fashion.

Reply:
https://github.com/dotnet/roslyn/wiki/NuGet-packages#versioning is always handy, btw.

Reply:
Yep, if 2.8.2 has 7.3 support then let's stay there.

Reply:
Done, building now.